### PR TITLE
Add ISO 19100 refs

### PR DESCRIPTION
**Pull request by dr-shorthair**
Initial selection from ISO 19100 series. 
Thanks to @andrea-perego for suggesting

### CODE REVIEW

**Comment by dr-shorthair** [comment]
Does this JSON file needed to be added to an inventory so it gets included in specref?

**Comment by dr-shorthair** [comment]
Oh dear - test failing, but does not appear to be in my new content :-(

File: refs/iso19100.json [hunk 1]
```diff
@@ -0,0 +1,293 @@
+{
+    "iso-19101-1": {"aliasOf": "iso-19101-1-2014"},
+    "iso-19101-1-2014": {
+        "authors": ["ISO/TC 211"],
+        "href": "https://www.iso.org/standard/59164.html",
+        "title": "Geographic information -- Reference model -- Part 1: Fundamentals",
+        "publisher": "ISO",
+        "deliveredBy": [{
```
Comment (tobie) [inline refs/iso19100.json]: I'd forgo deliveredBy altogether unless you have a use case for it.

Comment (dr-shorthair) [reply refs/iso19100.json]: The "deliveredBy" model is different in reference.json (list of url, shortname pairs) compared with raw-reference.json (list of URLs)  - is this the problem?

Our intention was to provide the link to the particular ISO Technical Committee which is responsible for all the ISO 19100 series standards. I think that is what deliveredBy is for?

File: refs/iso19100.json [hunk 2]
```diff
@@ -0,0 +1,249 @@
+{
```
Comment (tobie) [inline refs/iso19100.json]: You want to put all manual entries in biblio.jon for now.

**Comment by dr-shorthair** [comment]
Re-based. 
Copied iso-191** refs into biblio.json (for now)

**Pull request merged**